.github/workflows/ci.yml

```diff
@@ -90,7 +95,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install --upgrade "tox<4" tox-gh-actions
+          python -m pip install --upgrade "tox<4" tox-gh-actions setuptools
```

Comment:
why `setuptools` was added?

Reply:
`setuptools` is no longer bundled with Python 3.12